src/SFA.DAS.EmployerIncentives.Domain/ApprenticeshipIncentives/ValueTypes/ReinstatedPendingPaymentAudit.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using SFA.DAS.EmployerIncentives.Abstractions.Domain;
+
+namespace SFA.DAS.EmployerIncentives.Domain.ApprenticeshipIncentives.ValueTypes
+{
+    public class ReinstatedPendingPaymentAudit : ValueObject
```

Comment:
This shouldn't be a domain object.  When a pending payment is reinstated you get back a PendingPayment object not a ReinstatedPendingPaymentAudit object.

Reply:
This is used by the audit repository, and follows the same structure as the audit repository for the employment checks - EmploymentCheckRequestAudit is also a value type, which is mapped to the EF model for persistence

Reply:
OK